.github/workflows/workflow.yml

```diff
@@ -201,7 +241,7 @@ jobs:
       # otherwise the gem build will fail because it is not a development instance
       - uses: actions/setup-ruby@v1
         with:
-          ruby-version: ${{ env.RUBY_VERSION }}
+          ruby-version: '2.6.8'
```

Comment:
Is there a reason for not using the `env.RUBY_VERSION`? This includes the `ruby-version` additions above.

Reply:
because I explicitly want MRI 2.6.8 here. I had to remove the .ruby-version file so that for the rspec tests it would even be allowed to run with JRuby 9.3.8.0, but in so doing I have to be explicit for the other uses. And I still want to use MRI for these both to make sure nothing changes in them, and also because MRI still boots much faster than JRuby, so for reasons when strict compatibility (just running cucumber, but the tests themselves run on JRuby inside OpenHAB; running rubocop, etc.) isn't necessary, it's still preferred.